tests/unit/tuf_repository_service_worker/models/targets/test_crud.py

```diff
@@ -211,13 +211,24 @@ def test_update(self, monkeypatch):
         assert fake_datetime.now.calls == [pretend.call()]
 
     def test_update_to_published(self, monkeypatch):
-
+        test_targets = ["path/file1", "path.file2"]
+        crud.models.RSTUFTargets = pretend.stub(
+            path=pretend.stub(
+                in_=pretend.call_recorder(lambda *a: test_targets)
+            ),
+            published="published",
```

Comment:
Nit: I think the `published` value should be `True` as in the `update_to_published` you are updating it to `True`.

Reply:
If I do it, I will need to change the assert by something less readable.

```python
        assert mocked_update.update.calls == [
            pretend.call({"published": True, "last_update": fake_time})
        ]
```
to
```python
        assert mocked_update.update.calls == [
            pretend.call({"True": True, fake_time: fake_time})
        ]
```

This is because we are not controlling/testing what is inside the SQLAlchemy.
Which one do you prefer?

Reply:
I prefer the first. I got what you do here.
By stubbing the value of `crud.models.RSTUFTargets.published` you actually replace what value will be placed when you access it and given that the `update` API call accepts a `Dict[str, Any]` it should be a string.